base/slicearray.jl

```diff
@@ -232,3 +259,37 @@ Base.@propagate_inbounds setindex!(s::Slices{P,SM,AX,S,N}, val, I::Vararg{Int,N}
     s.parent[_slice_index(s, I...)...] = val
 
 parent(s::Slices) = s.parent
+
+function _element_size(s::Slices)
+    long = map((n,l) -> l === (:) ? n : nothing, size(s.parent), s.slicemap)
+    filter(!isnothing, long)
+end
+
+# These control summary printing, like `3-element eachcol(adjoint(::Matrix{Int64})) of ...`
+function showarg(io::IO, s::ColumnSlices, toplevel)
```

Comment:
```suggestion
function showarg(io::IO, s::ColumnSlices, toplevel::Bool)
```

Reply:
While it must be Bool, I followed `methods(Base.showarg)` in leaving it untyped.